web-ui/src/api/feedback.js

```diff
@@ -118,6 +119,44 @@ export const cancelFeedbackRequest = async (feedbackRequest, cookie) => {
   });
 };
 
+export const denyFeedbackRequest = async (requestId, reason, denier, creator, cookie) => {
+  console.log('Sending deny feedback request with data:', {
+    reason,
+    denier,
+    creator
+  });
+  return resolve({
+    method: 'POST',
+    url: `${feedbackRequestURL}/${requestId}/deny`,
+    data: { 
+      reason: reason, 
+      denier: denier, 
+      creator: creator,
+    },
+    headers: {
+      'X-CSRF-Header': cookie,
+      Accept: 'application/json',
+      'Content-Type': 'application/json;charset=UTF-8'
+    }
+  });
+};
+
+export const sendNotification = async (userId, message, cookie) => {
```

Comment:
```notifications.js``` already exists, and contains ```sendReminderNotification```. Therefore, ```sendNotification``` should remain in feedback.js until other use cases for on-platform notifications arise, so that the functions can remain separate for clarity. The existing ```notifications.js``` should probably be titled ```email.js``` instead. Let me know what you think.